drivers/at86rf215/Kconfig

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_AT86RF215
+    bool "Configure AT86RF215 driver"
+    depends on MODULE_AT86RF215
+    help
+        Configure the AT86RF215 driver using Kconfig.
+
+if KCONFIG_MODULE_AT86RF215
+
+config AT86RF215_USE_CLOCK_OUTPUT
+    bool "Enable/Disable Onchip Clock Source"
+    default n
+    help
+        Enable this to set the clock output of the AT86RF215
+        as the clock source on the board.
+        By Default it is turned off to save energy.
+
+config AT86RF215_TRIM_VAL
+    int "Trim value for the Crystal Oscillator"
+    range 0 15
+    default 0
+    depends on AT86RF215_USE_CLOCK_OUTPUT
```

Comment:
Does it really depend on this?

Reply:
Unless clock is enabled there is no real purpose to trim the clock.

Reply:
No, trim is used to make sure the send / receive frequency matches a common reference.
To do the trimming, you have to enable clock output so you can measure the actual frequency, then apply trim until it matches the expected frequency.

Ideally this would be done in an automated way during production and the resulting trim value would then be stored in some OTP area or EEPROM - but we are not quite there yet.

The at86rf215 requires an external 26 MHz oscillator, so if you want to save on BOM cost you can also use that the clock output of the radio as clock input for your MCU.

Reply:
So you are saying that initially there is dependency and once its calibrated the trim can be left as it is right ?

Reply:
Yes clock output is only needed for initial calibration (if it's not used otherwise), once you know the required trim value it can be disabled again.

Reply:
Thanks for the information. I have updated the file. [30a185c](https://github.com/RIOT-OS/RIOT/pull/13885/commits/30a185c5f49ba1eb0a265f3a8c10f8e247bba12a)

Reply:
@leandrolanzieri  Please see the above comment. It seems like the trim value is usually stored in an external memory. The trim value provided in the driver is for testing, if I am not wrong.

Reply:
> @leandrolanzieri Please see the above comments. It seems like the trim value is usually stored in an external memory. The trim value provided in the driver is for testing, if I am not wrong.

What's your point here?

The driver so far provides no direct way of setting this value for a particular hardware other than this parameter.

Reply:
@leandrolanzieri I understand that you were talking about make enabling the trim independently . What  I understood so far is
1. Trim is expected to work with the clock initially (mostly for calibration).
2. Default value 0 corresponds to 0pF which I believe is Zero trim. So do we need an external symbol ?